browser/src/control/Control.JSDialog.js

```diff
@@ -145,6 +145,7 @@ L.Control.JSDialog = L.Control.extend({
 				console.warn('closePopover: no builder');
 		}
 		else {
+			JSDialog.CloseDropdown(id);
```

Comment:
it calls: `L.Map.THIS.fire('jsdialog'` which probably will land here again.
easy to do a cycle, does it work? do we have cypress test for closing popup using that?

Reply:
No, I do not have evidence failing unit test